tests/acceptance/features/apiOcm/share.feature

```diff
@@ -1297,3 +1297,36 @@ Feature: an user shares resources using ScienceMesh application
       | Viewer          |
       | Uploader        |
       | Editor          |
+
+  @issue-10272
+  Scenario: federated user downloads shared resources as an archive
```

Comment:
does this scenario fit in this dir tests/acceptance/features/apiDownloads/

Reply:
I guess not because this is a `ocm` test and the federated user is downloading shared resources.(Just my opinion)

Reply:
tests/acceptance/features/apiDownloads/  contains tests scenario related to download despite of services so why you think that it doesn't fit there i can see spaceDownload too

Reply:
Sure. Will consider it if any more tests related to ocm download are to be added in near future.